### ISSUE TITLE
Wind electricity supply per neighborhood modelleeren

### ISSUE THREAD

**Issue by petertue** [ZEnMo/Digital-twin-Drechtsteden#7]
Ik heb een variabele aangemaakt in GZ_neighborhood: windElectricitySupply. Maar is nu nog een dummy variabele. Moet nog windgedrag krijgen.

**Comment by naudloomans** [comment]
Heb je het hier over wind kleinschalig? Wind grootschalig gaat namelijk doorgaans direct naar HSMS dus heeft weinig nut om per buurt te bekijken

**Comment by petertue** [comment]
Ja, is voorlopig nog niet zo relevant. Misschien als er bedrijventerreinen komen met kleinschalig wind

**Comment by naudloomans** [comment]
Aantal kleine windturbines (~25kW) max paar 1000 verdeeld random over buurten die bedrijventerrein of landbouw zijn (Hoe deze buurten automatisch te herkennen nog te onderzoeken)
